bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/values/connector/Executor.java

```diff
@@ -65,10 +65,10 @@ public static void submit(Scheduler scheduler, ObjectValue service, String resou
                               CallableUnitCallback callback, Map<String, Object> properties, Object... args) {
 
         //TODO Remove null check once scheduler logic is migrated for WebSocket. Scheduler cannot be null
-        if (scheduler == null) {
-            scheduler = new Scheduler(4, false);
-            scheduler.start();
-        }
+//        if (scheduler == null) {
```

Comment:
Shall we keep this commented code until we remove above submit API

Reply:
Added again https://github.com/ballerina-platform/ballerina-lang/pull/16116/commits/6d504a73e83a547ad3c00c91915d31dfc53edd69